src/schema/schema.ts

```diff
@@ -117,8 +117,14 @@ export function createZimbraSchema(
 				getAvailableLocales: (_: any) => client.getAvailableLocales(),
 				getMailboxMetadata: (_: any, variables) =>
 					client.getMailboxMetadata(variables as GetMailboxMetadataOptions),
-				getMessage: (_, variables) =>
-					client.getMessage(variables as GetMessageOptions),
+				getMessage: (_, variables, context = {}) => {
+					const { local } = context;
```

Comment:
I'm still nervous about building on this pattern because this this schema will not exist when we cut over to a real graphql endpoint. If we keep going with this pattern it is tech debt waiting to happen.

We have to move all local operations into `@client` resolvers instead of regular resolvers.

Reply:
Sure Peter, will look at how we can achieve it without creating duplicate queries - one with @client and one without it. That's the only part which is preventing us to move to client resolvers I think.

Reply:
@pl12133 I will set up a call to discuss this in detail with you, we would finalise a strategy and handle this before we merge this code to develop (currently it would go to ZAPPD-38 branch)